js/deer-render.js

```diff
@@ -204,6 +204,255 @@ DEER.TEMPLATES.shadow = (obj, options = {}) => {
+                            method = "PUT"
+                            commentAnno["@id"] = obj.comment.source.citationSource
+                        }
+                        fetch(url, {
+                            method: method,
+                            headers: {
+                                "Content-Type": "application/json",
+                                "Authorization": `Bearer ${DLA_USER.authorization}`
+                            },
+                            body: JSON.stringify(commentAnno)
+                        })
+                        .then(response => response.json())
+                        .then(anno => {
+                            statusComment.querySelector("span").innerHTML = "See your status comment"
+                            statusComment.querySelector("pre").innerHTML = elem.querySelector("textarea").value
+                            addRecordToManagedList(obj, elem, coll)
+                        })
+                        .catch(err => { })
+                        
+                    })
+
+                    elem.querySelector(".statusCommentText").addEventListener("keydown", e => {
+                        if( e.keyCode === 27 ) {
+                            e.target.value = ""
+                            document.getElementById("managedStatus").click()
+                        }
+                    })
+                }
+            })
+            .catch(err => { })
+
+            function addRecordToManagedList(obj, elem, coll){
+                //PARANOID CHECK to see if it is already in there
+                if(coll.itemListElement.filter(record => record["@id"] === obj["@id"]).length === 0){
+                    coll.itemListElement.push({"label": obj.label.value, "@id":obj["@id"]})
+                    coll.numberOfItems++
+                    fetch(DEER.URLS.OVERWRITE, {
+                        method: "PUT",
+                        mode: 'cors',
+                        headers: {
+                            "Content-Type": "application/json",
+                            "Authorization": `Bearer ${DLA_USER.authorization}`
+                        },
+                        body: JSON.stringify(coll)
+                    })
+                    .then(r => r.ok ? r.json() : Promise.reject(Error(r.text)))
+                    .then(data => {
+                        elem.innerHTML = `✔ Submitted For Review`
+                        // managedStatus.classList.add("is-hidden")
+                        // publicStatus.classList.add("is-hidden")
+                        //statusComment.settAttribute("deer-id", "")
+                        //statusComment.settAttribute("deer-id", obj["@id"])
+                        elem.classList.remove("bg-error")
+                        elem.classList.add("bg-success")
+                        setTimeout(() => UTILS.broadcast(undefined, DEER.EVENTS.NEW_VIEW, document, elem.querySelector(DEER.VIEW)), 0)
```

Comment:
What's the redraw for here?

Reply:
oo probably extraneous.  I had to redraw certain views b/c they had deer views in them.  I don't think this is one of those situations though.